### ISSUE TITLE
Consider using MeBots to allow users to add this bot to GroupMe chats automatically

### ISSUE THREAD

**Issue by ErikBoesen** [reteps/groupme-crosscountry-bot#1]
Hey there! I'm Erik, a developer of [another Python GroupMe bot](https://github.com/ErikBoesen/Yalebot). Motivated by frustration I experienced with building that bot, I recently developed [MeBots](http://mebots.co), which is a platform/API for building GroupMe bots more easily. It's designed to abstract away some of the headaches of maintaining GroupMe bots so that your job as a developer will be a little easier. For example, it has a whole built-in panel for adding and removing bots from different groups, and then it stores all the bot IDs and stuff in a database for you so you won't need to worry about keeping track of that data; you can just get what you need through a web API. Also, other people can add the bot to their own groups if you so choose. I think your bot could be of available to more people if it's on that marketplace and can be easily added by non-technical users.

I've written up some documentation for how to integrate with the API here: http://mebots.co/documentation. This would not be a complex change: each time a message was sent to a chat, there would just be one simple API request to check which bot_id to send with, and the website takes care of all the adding/removing logic for you!

I wrote some more specific documentation in the README of a pip package I made for easy integration with the API, which you can find [here](https://github.com/ErikBoesen/mebots.py), and there's a complete bot implementation [here](https://github.com/ErikBoesen/mebots-example-python)! I'm happy to make a PR integrating with the platform as well or answer any questions you may have about tradeoffs.

**Comment by reteps** [comment]
@ErikBoesen That would be awesome! I would love to contribute to your project. Can you open a PR with how I would end up integrating my bot?

**Comment by ErikBoesen** [comment]
Yeah, happy to!

**Comment by ErikBoesen** [comment]
Oh hey, I looked at your profile and I see you're on an FRC team! I just graduated from 1418. Best time of my life. Also, if I had gone to UMichigan I would have been the fourth generation in my family to do so! Love Ann Arbor.

**Comment by ErikBoesen** [comment]
Are you using Heroku to host this bot right now? If so I would recommend you use environment variables rather than statically setting things in your code: https://devcenter.heroku.com/articles/config-vars

**Comment by reteps** [comment]
good point, maybe i can fix that for my template. my other one is private so idk if it matters that much

**Comment by ErikBoesen** [comment]
I really would recommend fixing it in the private one as well!
